website/content/docs/plugins/index.mdx

```diff
@@ -7,12 +7,11 @@ description: Learn about Vault's plugin system.
 
 # Plugin System
 
-All Vault auth methods and secrets engines are considered plugins. This simple concept
-allows both built-in and external plugins to be treated like Legos. Any plugin
-can exist at multiple different locations. Different versions of a plugin may
-be at each location, with each version differing from Vault's version.
-
-@include 'plugin-file-permissions-check.mdx'
+All Vault auth methods and secrets engines are considered plugins. This simple
```

Comment:
Hi there. I suggest removing the adjective 'simple'; we don't want to make these types of implications in our docs and I believe is discouraged in our style guide. Also, same goes for 'Legos.' Instead of Legos, perhaps 'building blocks' would be a better reference?

Reply:
will do! Thanks!

Reply:
done in https://github.com/hashicorp/vault/pull/16923/commits/bc91187d2eeb3188bbac71b0fb6681c037991953